Test/GetMinuteQuarterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once "./src/MyClass.php";
+class GetMinuteQuarterTest extends TestCase
+{
+    public $obj;
+
+    
+    /**
+     * @dataProvider positiveDataProvider
+     */
+    public function testPositive($minute, $expected)
```

Comment:
I would suggest naming for this method - `testGetMinuteQuarterPositive`, because usually one Test class in unit testing, tests also one class